express-101/express101.js

```diff
@@ -1,8 +1,10 @@
+const express = require('express')
```

Comment:
Shouldn't the semicolons be added throughout the file? You won't have to worry about the 'Automatic Semicolon Insertion' and also it would prevent bugs.

Reply:
Yeah, I forgot here 😅